src/test/resources/testData/completion/enum_parameters_in.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
oh well, changing only the case is a problem for git+idea+mac, never knew that. 
will have to rename ```testData.completion``` -> ```testing.completion```, and ```mediaType_*``` -> ```media_type_*```